frontend/src/views/Login.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="login">
+    <Form @submit="submit">
```

Comment:
```suggestion
    <Form @submit.prevent="submit">
```
`e.preventDefault()`보다 조금 더 명시적이라고 생각됩니다 :)